code/modules/fishing/sources/_fish_source.dm

```diff
@@ -190,8 +190,7 @@ GLOBAL_LIST(fishing_property_cache)
 
 		final_table[result] *= rod.multiplicative_fish_bonus(result, src)
 		final_table[result] += rod.additive_fish_bonus(result, src) //Decide on order here so it can be multiplicative
-		if(result == FISHING_DUD)
-		else if(ispath(result, /obj/item/fish))
```

Comment:
how does this not fail any sort of linter

Reply:
'Cause we don't don't have a linter that discriminate empty control statements, perhaps.